service/kernel/error_handling.cpp

```diff
@@ -39,20 +39,6 @@ namespace services
     this->_errors->add(services::ErrorIncorrectNumberOfElementsInResultCollection);
 */
 
-namespace internal {
-Status checkForNullByteInjection(const char *begin, const char *const end)
```

Comment:
FYI
Ideally it should be solved by adding DAAL_EXPORT define to the function in .cpp file, moving implementation to header file is not the best way.